acceptance-tests/src/test/java/gov/di_ipv_drivingpermit/pages/WelshLangDrivingLicencePageObject.java

```diff
@@ -0,0 +1,1015 @@
+        Assert.assertEquals(expectedText, actualText);
+        LOGGER.info(actualText);
+    }
+
+    public void licenceNumberWelshDVA() {
+        String expectedText = "Rhif trwydded";
+        String actualText = licenceNumberFieldDVA.getText();
+        Assert.assertEquals(expectedText, actualText);
+        LOGGER.info(actualText);
+    }
+
+    public void licenceSentence() {
+        String expectedText =
+                "Dyma'r rhif hir yn adran 5 ar eich trwydded er enghraifft HARRI559146MJ931";
+        String actualText = licenceNumberExampleField.getText();
+        Assert.assertEquals(expectedText, actualText);
+        LOGGER.info(actualText);
+    }
+
+    public void issueNumberWelsh() {
+        String expectedText = "Rhif cyhoeddi";
+        String actualText = issueNumberField.getText();
+        Assert.assertEquals(expectedText, actualText);
+        LOGGER.info(actualText);
+    }
+
+    public void issueNumberSentence() {
+        String expectedText = "Dyma'r rhif 2 ddigid ar ôl y gofod yn adran 5 o'ch trwydded";
+        String actualText = issueeNumberExampleField.getText();
+        Assert.assertEquals(expectedText, actualText);
+        LOGGER.info(actualText);
+    }
+
+    public void noDrivingLicenceBtnWelsh() {
+        noDLOption.isDisplayed();
+        noDLRadioBtn.isDisplayed();
+    }
+
+    public void orDisplayWelsh() {
+        orOption.isDisplayed();
+    }
+
+    public void postcodeWelsh() {
+        String expectedText = "Cod post";
+        String actualText = postCodeField.getText();
+        Assert.assertEquals(expectedText, actualText);
+        LOGGER.info(actualText);
+    }
+
+    public void postcodeSentence() {
+        String expectedText = "Rhowch y cod post yn y cyfeiriad yn adran 8 o'ch trwydded";
+        String actualText = postCodeExampleField.getText();
+        Assert.assertEquals(expectedText, actualText);
+        LOGGER.info(actualText);
+    }
+
+    public void continueButtonWelsh() {
+        CTButton.isDisplayed();
+        CTButton.isEnabled();
+    }
+
+    public void licenceSelectionSentence() {
+        if (Driver.get()
+                .getPageSource()
+                .contains(
+                        "Gallwch ddod o hyd i hwn yn adran 4c o'ch trwydded yrru. Bydd naill ai'n dweud DVLA (Asiantaeth Trwyddedu Gyrru a Cherbydau) neu DVA (Asiantaeth Gyrrwyr a Cherbydau).")) {
+            LOGGER.info(
+                    "lastname is \"Gallwch ddod o hyd i hwn yn adran 4c o'ch trwydded yrru. Bydd naill ai'n dweud DVLA (Asiantaeth Trwyddedu Gyrru a Cherbydau) neu DVA (Asiantaeth Gyrrwyr a Cherbydau).\" ");
+        }
+    }
+
+    public void validToDateFieldTitle() {
+        String expectedText = "Yn ddilys tan";
+        String actualText = ValidToDatefieldWelsh.getText();
+        Assert.assertEquals(expectedText, actualText);
+        if (Driver.get().getPageSource().contains("Yn ddilys tan")) {
+            LOGGER.info(" Issue date as \"Yn ddilys tan\" ");
```

Comment:
Same here should this have an assert

Reply:
Changed to Assert.assertEquals(expectedText, actualText)